pipelines/release/codekit/github_tag_release.groovy

```diff
@@ -1,66 +0,0 @@
-node('jenkins-manager') {
```

Comment:
Can't remove this one.

Reply:
Actually, it looks like this pipeline is not used, as the release jobs invoke the same code directly, not via the pipeline.  Removal is OK.